xgcm/grid.py

```diff
@@ -1121,6 +1138,8 @@ def interp(self, da, axis, **kwargs):
         Parameters
         ----------
         %(grid_func.parameters)s
+        keep_coords : boolean, optional
```

Comment:
Sorry for catching this now. Could you add this in the docstring of `grid_func`, that will get parsed here.